tools/build/scanCode.cfg

```diff
@@ -0,0 +1,54 @@
+# scanCode.py configuration file
+
+# List of filenames containing the text of valid license (headers)
+# These files SHOULD be in the same directory path where scanCode.py
+# resides.
+[Licenses]
+ASFLicenseHeader.txt
+ApacheIBMLicenseHeader.txt
+
+# Filters (path/filename) with wildcards and associated scan checks
+# that are to be run against them.  The checks are actual valid
+# function names found in scanCode.py.
+[Includes]
+*=is_not_symlink
+*.scala=has_block_license, no_tabs, no_trailing_spaces, eol_at_eof
+*.py=no_tabs, no_trailing_spaces, eol_at_eof
+*.java=has_block_license, no_tabs, no_trailing_spaces, eol_at_eof
+*.js=no_tabs, no_trailing_spaces, eol_at_eof
+*.gradle=no_tabs, no_trailing_spaces, eol_at_eof
+*.md=no_tabs, eol_at_eof
+*.go=has_block_license, no_tabs, no_trailing_spaces, eol_at_eof
+build.xml=no_tabs, no_trailing_spaces, eol_at_eof
+deploy.xml=no_tabs, no_trailing_spaces, eol_at_eof
+
+# List of paths (inclusive of subdirectories) to exlude from code scanning
+[Excludes]
+# General exclusions
+.tox
+.git
+# OpenWhisk exclusions
+bin
+tests/build/reports
+tests/dat
+# openwhisk-catalog exclusions (Python samples)
+packages/samples/hello
+# The following repos. have so far been identified as having scanning errors
+# and will be excluded until their owners have completed updates to add Travis
```

Comment:
@rabbah as part of my testing this helped me identify which repos are not currently running scans at all and reporting, in some cases, 100s of errors each.  It was my hope that we open issues for each to support scanCode.py in their Travis using the method the 5 repos that now do scan by pulling in openwhisk do (by reference) (note that most do not even have Travis) then open other issues to fix their scanCode errors and then uncomment them in .cfg in openwhisk.  It was a planned progression to get all repos. on board using the same config.  Although each could supply their own config, it would be desirable IMO if all would use a project-wide set of checks (or we might see repo. owners start removing checks.  The initial goal is to assure all code, in all project repos., have proper ASF license headers.  Just my approach.